target/scripts/start-mailserver.sh

```diff
@@ -256,9 +256,8 @@ function _defunc
 # ------------------------------------------------------------
 
 _notify 'tasklog' "Welcome to docker-mailserver $(</VERSION)"
-_notify 'inf' 'ENVIRONMENT'
-[[ ${DMS_DEBUG} -eq 1 ]] && printenv
 
+[[ ${DMS_DEBUG} -eq 1 ]] && print_environment
```

Comment:
What about not printing the env on startup but making this a callable script that can be executed if needed (e.g. in the form when it comes to the environment make a hint to run `docker exec -it mailserver print_environment` to get the env on runtime.

I see multiple benefits on this way as ENVs might getting changed after startup or you do not want to restart your server to fill an issue report (and may loose the actual flawed instance with logs)

Reply:
I implemented this behavior. But we will now need to use the `.bashrc` file because we do not have the `${VARS}` array at hand. Done with 4255affabfa916bb4a38f490eb93e907ecc6b2e5.

Reply:
Here you go:

https://github.com/docker-mailserver/docker-mailserver/blob/6336c0b256682527b880032301da4d3ba555e030/target/scripts/startup/setup-stack.sh#L70-L83

Reply:
1. `docker exec mailserver cat /root/.bashrc` to get also the container ENVs
2. ENVs should not change during runtime, do they?
3. It doesn't hurt IMO, when `DMS_DEBUG` outputs the configured ENVs. PS: There might be people, letting DMS_DEBUG configured all the time and not only when there are issues 😉

Reply:
Ah thanks for clarification 👀 now i get it

Reply:
Do I overlook something? ATM `$DMS_DEBUG=1` does print the environment variables and we have a script that does this so we're able to execute it at a later point in time too. Isn't this the best of both worlds?

Reply:
Yep. The only thing is, do we need an additional script, when `docker exec mailserver cat /root/.bashrc` provides the same information? (PS: I am also fine, if we keep the script 😉)

Reply:
I think this is the way to go, because `cat`ing the `.bashrc` does not format the chosen ENV variables as nice as my little addition does 😆